README.md

```diff
@@ -356,6 +356,96 @@ countItems := response.CountItems
 count := response.Count
 ```
 
+## Importing and Exporting Data
+
+### Importing Data
+
+If you have large amounts of data, it is more efficient to import it to Pilosa instead of several `SetBit` queries. This library supports importing bits of a frame. The target frame must have been created before hand.
+
+Before starting the import, create an instance of a struct which implements `BitIterator` and pass it to `client.ImportFrame` function. This library ships with `CSVBitIterator` struct which supports importing bits in the CSV (comma separated values) format:
+```
+ROW_ID,COLUMN_ID
+```
+
+Optionally, a timestamp with GMT time zone can be added:
```

Comment:
UTC is equivalent to GMT, but it seems UTC is the newer name, so I will use it instead. Most of the time the programmer has to select between getting the timestamp in local time or UTC, so I think we should have a convention.